.github/workflows/build-pr.yml

```diff
@@ -51,8 +51,9 @@ jobs:
       - name: Python Setup
         working-directory: cla-backend
         run: |
+          pip install --platform manylinux2014_x86_64 --only-binary=:all: --target=/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages cryptography    
```

Comment:
create another requirements file such that theres backward compatability. You can create requirements-docusign.txt with the docusign-esign==3.22.0 package and then add the pip install command for this specific file  ie `pip install --platform manylinux2014_x86_64 --platform manylinux2014_x86_64 --only-binary=:all: --target=/opt/hostedtoolcache/Python/3.7.17/x64/lib/python3.7/site-packages cryptography -r requirements-docusign.txt`

Reply:
@umeshlumbhani247 I have updated the code, please review the same,